src/components/CollectionLink.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+export default class CollectionLink extends React.Component<any, any> {
+  constructor(props: CollectionLinkProps) {
+    super(props);
+  }
+
+  render(): JSX.Element {
+    return (
+      <a className={this.props.className} href={"?url=" + this.props.url} onClick={this.handleClick.bind(this)}>
+        {this.props.text || this.props.children}
+      </a>
+    );
+  }
+
+  handleClick(event) {
+    // if any of these keys are pressed, the url is opened by the browser as it pleases
+    if (event.metaKey || event.altKey || event.ctrlKey) {
```

Comment:
:+1: and let's also add shift just in case. In chrome, shift click opens a new window instead of a new tab.